source/sass/buyers-guide/views/catalog.scss

```diff
@@ -215,6 +215,9 @@ body.catalog {
         $recommendation-width: 17px;
         $recommendation-height: 15.8px;
 
+        // Needed for animation state while JS loads
+        opacity: 0;
+
```

Comment:
This will also mean that people with JS disabled see nothing, not even a "please turn on javascript to use this guide", which is not great: is there a way we can make sure this property gets preset only when JS is enabled?